modules/gltf/gltf_document.cpp

```diff
@@ -123,18 +123,25 @@ static Ref<ImporterMesh> _mesh_to_importer_mesh(Ref<Mesh> p_mesh) {
 	for (int32_t surface_i = 0; surface_i < p_mesh->get_surface_count(); surface_i++) {
 		Array array = p_mesh->surface_get_arrays(surface_i);
 		Ref<Material> mat = p_mesh->surface_get_material(surface_i);
+		String surface_name = array_mesh.is_valid() ? array_mesh->surface_get_name(surface_i) : "";
```

Comment:
```suggestion
		String surface_name = array_mesh.is_valid() ? array_mesh->surface_get_name(surface_i) : String();
```

Reply:
I'll apply these to the previous PR that this one requires